### PR TITLE
Support for IPAM to nmstate

### PR DESCRIPTION
**Pull request by anmazzotti**
Support for https://github.com/rancher/elemental-operator/pull/793

### CODE REVIEW

File: Dockerfile.dev.os [hunk 1]
```diff
@@ -56,6 +59,13 @@ RUN ARCH=$(uname -m); \
       dmidecode \
       libopenssl1_1
 
+# Install nmstatectl
+RUN wget https://github.com/nmstate/nmstate/releases/download/v2.2.33/nmstatectl-linux-x64.zip && \
```
Comment (kkaempf) [inline Dockerfile.dev.os:63]: Uh, oh, we rather should get this packaged

**Review by kkaempf — APPROVED** [review]
worth a try

**Comment by anmazzotti** [comment]
Sorry this was not meant to be ready for review. Back to draft.

File: Dockerfile.dev.os (hunk 1 above)
Comment (kkaempf) [reply Dockerfile.dev.os:63]: I'm trying to package it ... 🤞🏻

**Review by fgiudici — APPROVED** [review]
lgtm

**Pull request merged**